### PR TITLE
renamed time-tracking and updated imports

### LINKED ISSUE #706: Update time-tracking folder to TimeTracking in src/components
**Bug:** 
currently `time-tracking` folder inside `app/javascript/src/components/` is not following our naming convention.

**Solution**
Rename the `time-tracking` folder to `TimeTracking` inside `app/javascript/src/components/` & update component import paths respectively.

### PR DESCRIPTION
**Pull request by camperjett**
fixes #706 
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [ ] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by camperjett** [comment]
@akhilgkrishnan PTAL

**Comment by akhilgkrishnan** [comment]
@prasanthchaduvula can you review this?

File: app/javascript/src/constants/routes.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import TimeEntryReports from "../components/Reports/timeEntry";
 import TotalHoursReport from "../components/Reports/totalHoursLogged";
 import PlanSelection from "../components/Subscriptions/PlanSelection";
 import TeamRouteConfig from "../components/Team/RouteConfig";
-import TimeTracking from "../components/time-tracking/Index";
+import TimeTracking from "../components/TimeTracking/Index";
```
Comment (prasanthchaduvula) [inline app/javascript/src/constants/routes.ts:22]: @camperjett No need of giving `Index`. Rename `Index` file to `index` inside `TimeTracking` folder and update `import TimeTracking from "../components/TimeTracking/Index" ` to import TimeTracking from "../components/TimeTracking";;

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

**Review by prasanthchaduvula — APPROVED** [review]

**Review by apoorv1316 — APPROVED** [review]

**Comment by supriya3105** [comment]
@camperjett Can you check the CI tests that are failing with this PR ?

**Comment by apoorv1316** [comment]
@camperjett Thanks for contributing to miru.

**Pull request merged**